embedded-graphics/src/draw_target/clipped.rs

```diff
@@ -6,6 +6,12 @@ use crate::{
 };
 
 /// Clipped draw target.
+///
+/// Created by calling [`clipped`] on any [`DrawTarget`].
+/// See the [`clipped`] method documentation for more.
+///
+/// [`DrawTarget`]: trait.DrawTarget.html
```

Comment:
Just making a note here since I was once asked to revert intra-crate links to regular ones, because the feature is not yet stable. Right now it's targeted at 1.47.0, which is still some time away. To be fair, with embedded-layout, I'm also using these with a note that although the crate's MSRV is 1.40 or so, the docs require nightly to build.